algebird-core/src/main/scala/com/twitter/algebird/CountMinSketch.scala

```diff
@@ -138,14 +139,14 @@ case class CMSAggregator[K](cmsMonoid: CMSMonoid[K]) extends MonoidAggregator[K,
  * @param eps One-sided error bound on the error of each point query, i.e. frequency estimate.
  * @param delta A bound on the probability that a query estimate does not lie within some small interval
  *              (an interval that depends on `eps`) around the truth.
+ * @param maxExactCount The largest number of keys that will be stored directly, instead of in a sketch.
  * @tparam K The type used to identify the elements to be counted.
  */
-case class CMSParams[K](hashes: Seq[CMSHash[K]], eps: Double, delta: Double) {
+case class CMSParams[K](hashes: Seq[CMSHash[K]], eps: Double, delta: Double, maxExactCount: Int = 0) {
```

Comment:
We should add a `require(maxExactCount >= 0, "maximum exact count must not be negative")`.